backend/src/database/models/message.ts

```diff
@@ -1,22 +1,29 @@
-import * as mongoose from 'mongoose'
+import * as mongoose from 'mongoose';
 
-const messageSchema = new mongoose.Schema({
-    sender_name: {
-        type: String,
-        required: true
+export interface MessageType {
+  senderName: string;
+  text: string;
+  date?: Date;
+}
+
+const messageSchema = new mongoose.Schema(
+  {
+    senderName: {
```

Comment:
[mongo powinno byc camelCase](https://intellipaat.com/community/28608/what-are-naming-conventions-for-mongodb)